docs/tools.html

```diff
@@ -219,8 +219,8 @@
 
```

Comment:
Noticed the vim plugin doesn't point to your new repo. Want to transfer to reasonml and we can update the docs for Vim too?